src/modules/commander/commander_params.c

```diff
@@ -912,3 +912,19 @@ PARAM_DEFINE_FLOAT(COM_KILL_DISARM, 5.0f);
  * @increment 1
  */
 PARAM_DEFINE_FLOAT(COM_CPU_MAX, 90.0f);
+
+/**
+ * Time-out for detecting a failure after takeoff
+ *
+ * A non-zero, positive value specifies the timeframe in seconds within Failure Detector is allowed to put the vehicle into
+ * a lockdown state if attitude exceeds the defined limits.
```

Comment:
This does not hold for all flight modes. Can you either clarify or better reference where it is clearly described?